resources/usa_injury/cl_injury.lua

```diff
@@ -1,176 +1,15 @@
-    [GetHashKey("WEAPON_BATTLEAXE")] = {type = 'laceration', bleed = 240, string = 'Large Sharp Object', treatableWithBandage = false, treatmentPrice = 300, dropEvidence = 1.0}, -- WEAPON_BATTLEAXE
-    [GetHashKey("WEAPON_POOLCUE")] = {type = 'blunt', bleed = 1800, string = 'Large Blunt Object', treatableWithBandage = false, treatmentPrice = 65, dropEvidence = 1.0}, -- WEAPON_POOLCUE
-    [GetHashKey("WEAPON_WRENCH")] = {type = 'blunt', bleed = 900, string = 'Concentrated Blunt Object', treatableWithBandage = false, treatmentPrice = 80, dropEvidence = 1.0}, -- WEAPON_WRENCH
-    [GetHashKey("WEAPON_FLASHLIGHT")] = {type = 'blunt', bleed = 2700, string = 'Blunt Object', treatableWithBandage = true, treatmentPrice = 40, dropEvidence = 0.6}, -- WEAPON_FLASHLIGHT
-    [GetHashKey("WEAPON_PISTOL_MK2")] = {type = 'penetrating', bleed = 300, string = 'High-speed Projectile', treatableWithBandage = false, treatmentPrice = 500, dropEvidence = 1.0}, -- WEAPON_PISTOL_MK2
-    [GetHashKey("WEAPON_CARBINERIFLE_MK2")] = {type = 'penetrating', bleed = 300, string = 'High-speed Projectile', treatableWithBandage = false, treatmentPrice = 500, dropEvidence = 1.0}, -- WEAPON_CARBINERIFLE_MK2
-    [GetHashKey("WEAPON_PUMPSHOTGUN_MK2")] = {type = 'penetrating', bleed = 300, string = 'High-speed Projectile', treatableWithBandage = false, treatmentPrice = 500, dropEvidence = 1.0}, -- WEAPON_PUMPSHOTGUN_MK2
-    [GetHashKey("WEAPON_ASSAULTRIFLE_MK2")] = {type = 'penetrating', bleed = 300, string = 'High-speed Projectile', treatableWithBandage = false, treatmentPrice = 500, dropEvidence = 1.0}, 
-    [GetHashKey("WEAPON_COMPACTRIFLE")] = {type = 'penetrating', bleed = 300, string = 'High-speed Projectile', treatableWithBandage = false, treatmentPrice = 500, dropEvidence = 1.0}, -- WEAPON_COMPACTRIFLE
-    [GetHashKey("WEAPON_M4GOLDBEAST")] = {type = 'penetrating', bleed = 300, string = 'High-speed Projectile', treatableWithBandage = false, treatmentPrice = 500, dropEvidence = 1.0},
-}
-
-hospitalLocations = {
-    {307.10046386719, -595.07073974609, 43.284019470215}, -- upper PB
-    {350.97692871094, -587.77874755859, 28.796844482422}, -- lower PB
-    {-817.61511230469, -1236.6121826172, 7.3374252319336}, -- viceroy medical
-    {1768.4294433594, 2570.2172851563, 45.729831695557}, -- prison
-    {1832.7521, 3677.0686, 34.2749}, -- sandy
-    {-251.8987, 6334.1558, 32.4272} -- Paleto Clinic
-}
-
-
 effects = {} -- when you take damage for a specific reason, you may be put into an effect
 injuredParts = {} -- injured body parts, and their wounds as the value
 
+local InjuriesC = Config.Injuries
+
 ------ NOTIFY PLAYER OF INJURIES ------
 
 RegisterNetEvent('injuries:showMyInjuries')
 AddEventHandler('injuries:showMyInjuries', function()
     local hadInjury = false
-    for bone, injuries in pairs(injuredParts) do
-        local boneName = parts[bone]
+    for bone, InjuriesC in pairs(injuredParts) do
```

Comment:
```suggestion
    for bone, injuryInfo in pairs(injuredParts) do
```
`InjuriesC` is already defined above (line 4)